pandas/core/indexes/interval.py

```diff
@@ -432,10 +432,8 @@ def closed(self):
     )
     def set_closed(self, closed):
         if closed not in _VALID_CLOSED:
-            msg = "invalid option for 'closed': {closed}"
-            raise ValueError(msg.format(closed=closed))
+            raise ValueError(f"invalid option for 'closed': {closed}")
 
-        # return self._shallow_copy(closed=closed)
```

Comment:
is there a reason to remove this comment?

Reply:
No. I put it back as it was placed.